packages/warriorjs-cli/src/ProfileGenerator.js

```diff
@@ -44,15 +44,32 @@ class ProfileGenerator {
    */
   generateReadmeFile() {
     const template = fs.readFileSync(README_TEMPLATE_FILE_PATH, 'utf8');
-    const data = {
-      getFloorMap,
-      getFloorMapKey,
-      profile: this.profile,
-      level: this.level,
-    };
+    const levels = [];
+
     const options = { filename: README_TEMPLATE_FILE_PATH };
-    const renderedReadme = ejs.render(template, data, options);
-    fs.writeFileSync(this.profile.getReadmeFilePath(), renderedReadme);
+    if (this.profile.epic) {
+      for (let i = 1; i < 10; i += 1) {
+        const data = {
+          getFloorMap,
+          getFloorMapKey,
+          profile: this.profile,
+          level: i,
```

Comment:
You're just passing the level number here, where we need the level config object instead. Check what we pass inside `this.level`, that should be what we pass for each level here. Also, check my general comment. The [commit](https://github.com/olistic/warriorjs/commit/f2db49051dd5fb06c6aac829acb30b066d2fd5ec) I talk about will make this much easier and cleaner.